apps/dashboard/src/components/header-navigation/no-changes-modal.tsx

```diff
@@ -14,27 +14,24 @@ export function NoChangesModal({ isOpen, onClose }: NoChangesModalProps) {
       <DialogContent className="max-w-md gap-4 p-3">
         <div className="flex items-start justify-start">
           <svg width="116" height="44" viewBox="0 0 116 44" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <rect x="0.5" y="0.5" width="115" height="43" rx="7.5" stroke="#E1E4EA" strokeDasharray="5 3" />
+            <rect x="0.5" y="0.5" width="115" height="43" rx="7.5" stroke="#E1E4EA" stroke-dasharray="5 3" />
```

Comment:
```suggestion
            <rect x="0.5" y="0.5" width="115" height="43" rx="7.5" stroke="#E1E4EA" strokeDasharray="5 3" />
```

This should be like this in react